samples/client/petstore/perl/README.md

```diff
@@ -510,7 +509,3 @@ Authentication schemes defined for the API:
 
 - **Type**: HTTP basic authentication
 
-## http_signature_test
-
-- **Type**: HTTP basic authentication
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect line of documentation

Reply:
now outdated